### ISSUE TITLE
Scheduler is not enabled for some content types

### ISSUE THREAD

**Issue by ddrozdik** [ymcatwincities/openy#2331]
While working with the latest version of Open Y (2.7.0 & 2.7.1) we noticed that Scheduler is not enabled for some content types while similar content types have the ability to schedule the date of publishing. 

Find below the list of content types that should support scheduling the date of publishing:
- Event 
- News Post
- Social Post

**Comment by podarok** [comment]
@ddrozdik is this a request from associations? @sarah-halby let's ensure this enhancement meets enough demand and we have the bandwidth for it. 
We also have some content types in Virtual Y and Membership Framework could be enabled for scheduling, but this is an enhancement.
Scheduler is shipped with Open Y so Content Managers can enable it easily without asking Open Y team to develop this in the core.

**Comment by ddrozdik** [comment]
@podarok, yes, we enable it for events and news posts almost every time. Since this is not a part of Open Y yet we have to treat and support such changes as customizations. 

But I would set the priority to minor since this is not a critically important thing however still would be nice to have in core.

**Comment by sarah-halby** [comment]
Sarah to reach out to IXM, OneEach, JWTech to determine how they approach enabling for events & news posts with the goal to determine whether we should include this in the core and/or create documentation on how to enable this yourself without adding to the core. FYI @ddrozdik
